test/init/generator/init-inquirer.test.js

```diff
@@ -22,16 +22,32 @@ describe('init', () => {
     });
 
     it('should scaffold when given answers', async () => {
-        const { stdout } = await runPromptWithAnswers(genPath, ['init'], [`N${ENTER}`, ENTER, ENTER, ENTER, ENTER, ENTER, ENTER]);
+        const { stdout } = await runPromptWithAnswers(genPath, ['init'], [`N${ENTER}`, ENTER, ENTER, ENTER, ENTER, ENTER]);
 
         expect(stdout).toBeTruthy();
         expect(stdout).toContain(firstPrompt);
 
+        // Skip test in case yarn error log is generated
+        if (fs.existsSync(resolve(genPath, './yarn.lock'))) {
```

Comment:
I assume you wanna check for the existence of `yarn-error.log`.

Reply:
Actually non existence of lock file.